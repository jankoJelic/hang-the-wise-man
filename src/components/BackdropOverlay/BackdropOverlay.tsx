type Props = {
  children: React.ReactNode;
  className?: string;
};
const BackdropOverlay: React.FC<Props> = ({ children, className }) => {
  return (
    <div
      className={`flex fixed inset-0 left-0 w-screen h-screen
   bg-mainText bg-opacity-60 items-center justify-center shadow-md ${className}`}
    >
      {children}
    </div>
  );
};

export default BackdropOverlay;
