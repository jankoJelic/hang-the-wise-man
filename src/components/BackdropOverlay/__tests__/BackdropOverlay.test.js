import renderer from "react-test-renderer";
import BackdropOverlay from "../BackdropOverlay";

it("renders correctly", () => {
  const tree = renderer.create(<BackdropOverlay />).toJSON();
  expect(tree).toMatchSnapshot();
});
