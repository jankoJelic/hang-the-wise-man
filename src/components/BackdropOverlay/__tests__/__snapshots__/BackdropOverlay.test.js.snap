// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex fixed inset-0 left-0 w-screen h-screen
   bg-mainText bg-opacity-60 items-center justify-center shadow-md undefined"
/>
`;
